tests/plugins/test_pull_base_image.py

```diff
@@ -533,15 +533,16 @@ def workflow_callback(workflow):
                                     check_platforms=True)
         assert log_message in caplog.text
 
-    def test_single_platform_build(self, caplog):
+    @pytest.mark.parametrize('fetch_manifest_list', (True, False))
+    def test_single_platform_build(self, caplog, fetch_manifest_list):
 
         def workflow_callback(workflow):
             workflow = self.prepare(workflow)
             workflow.prebuild_results[PLUGIN_CHECK_AND_SET_PLATFORMS_KEY] = set(['x86_64'])
             return workflow
 
-        log_message = 'single platform build'
-        test_pull_base_image_plugin(LOCALHOST_REGISTRY, BASE_IMAGE,
+        log_message = 'Base image is a manifest list for all required platforms'
```

Comment:
this should be based on 'fetch_manifest_list' and if that is False
log_message should be : 
'Skipping validation of available platforms for base image: '
'this is a single platform build and base image has no manifest '
'list'